### PR TITLE
Ruby on Rails: Add descriptive link text

### PR DESCRIPTION
**Pull request by guskirb**
## Because

Provide descriptive link text to "APIs and Building Your Own" lesson and fix lint errors.

## This PR

* Add descriptive link text to "APIs and Building Your Own" lesson.
* Fix lint errors.

## Issue

Related to #28290

## Pull Request Requirements

-   [x] I have thoroughly read and understand [The Odin Project curriculum contributing guide](https://github.com/TheOdinProject/curriculum/blob/main/CONTRIBUTING.md)
-   [x] The title of this PR follows the `location of change: brief description of change` format, e.g. `Intro to HTML and CSS lesson: Fix link text`
-   [x] The `Because` section summarizes the reason for this PR
-   [x] The `This PR` section has a bullet point list describing the changes in this PR
-   [x] If this PR addresses an open issue, it is linked in the `Issue` section
-   [x] If any lesson files are included in this PR, they have been previewed with the [Markdown preview tool](https://www.theodinproject.com/lessons/preview) to ensure it is formatted correctly
-   [x] If any lesson files are included in this PR, they follow the [Layout Style Guide](https://github.com/TheOdinProject/curriculum/blob/main/LAYOUT_STYLE_GUIDE.md)

### CODE REVIEW

File: ruby_on_rails/apis/apis_and_building_your_own.md [hunk 1]
```diff
@@ -192,30 +191,32 @@ In the next lesson, we'll cover working with other people's APIs, which can add
...
 ### Additional resources
 
-This section contains helpful links to related content. It isn’t required, so consider it supplemental.
+This section contains helpful links to related content. It isn't required, so consider it supplemental.
 
-- Watch [this free Railscast on making your App into an API](http://railscasts.com/episodes/348-the-rails-api-gem)
-- Watch [this free Railscast on securing your API](http://railscasts.com/episodes/352-securing-an-api)
-- Watch [this free Railscast on versioning your API](http://railscasts.com/episodes/350-rest-api-versioning)
+- Watch this free [Railscast on making your App into an API](http://railscasts.com/episodes/352-securing-an-api)
+- Watch this free [Railscast on versioning your API](http://railscasts.com/episodes/352-securing-an-api)
```
Comment (wise-king-sullyman) [inline ruby_on_rails/apis/apis_and_building_your_own.md:218]: Looks like the same link got repeated here twice and the description got off on the second one.

**Review by wise-king-sullyman — CHANGES_REQUESTED** [review]
This looks great, thanks!

Just a couple small things I noticed with the additional resources.

File: ruby_on_rails/apis/apis_and_building_your_own.md (hunk 1 above)
Comment (MaoShizhong) [reply ruby_on_rails/apis/apis_and_building_your_own.md:218]: That'll be my fault. Botched the copy+paste for a few cells in the spreadsheet (which have now been fixed :eyes:)

Comment (guskirb) [reply ruby_on_rails/apis/apis_and_building_your_own.md]: Replaced them 👍🏻

**Review by MaoShizhong — APPROVED** [review]
:pray:

**Comment by MaoShizhong** [comment]
@wise-king-sullyman Just noticed you had requested changes, so can't merge this myself yet! :sweat_smile:

**Review by wise-king-sullyman — APPROVED** [review]
Looks awesome, thanks!

**Pull request merged**